basket/news/country_codes.py

```diff
@@ -252,7 +252,7 @@
   'zm': 'Zambia',
   'zw': 'Zimbabwe',
 }
-SFDC_COUNTRIES_LIST = SFDC_COUNTRIES.keys()
+SFDC_COUNTRIES_LIST = list(SFDC_COUNTRIES.keys())
```

Comment:
Does this need to be converted to a list because it's exported/imported?

Reply:
Dict functions like `keys()` and `items()` return an iterator in Python 3 where they used to return a `list` in Python 2. The 2to3 tool did this just to make sure it all worked the way it used to. I removed a lot of those uses of `list()` for this PR but left that one since I wasn't immediately sure how it was used. It would probably be fine to not cast it here, but it also doesn't hurt.